odo/backends/sql.py

```diff
@@ -103,7 +103,7 @@
     mssql.UNIQUEIDENTIFIER: string,
     # The SQL Server TIMESTAMP value doesn't correspond to the ISO Standard
     # It is instead just a binary(8) value with no relation to dates or times
-    mssql.TIMESTAMP: bytes_,
+    # mssql.TIMESTAMP: bytes_,
```

Comment:
Ah ok I understand. Yea this is weird but the issue is probably because of this:

```
In [21]: isinstance(sa.TIMESTAMP(), mssql.TIMESTAMP)
Out[21]: True
```
I think we would've expected `mssql.TIMESTAMP` to be a child of `sa.TIMESTAMP` but that does not appear to be the case as can be seen here: https://github.com/zzzeek/sqlalchemy/blob/master/lib/sqlalchemy/dialects/mssql/base.py#L916